codify/deployment.go

```diff
@@ -28,36 +28,40 @@ import (
 	"fmt"
 	"github.com/kris-nova/logger"
 	appsv1 "k8s.io/api/apps/v1"
+	"strings"
 	"text/template"
 	"time"
 )
 
 type Deployment struct {
-	i *appsv1.Deployment
+	*appsv1.Deployment
```

Comment:
I can see how this makes the embedded deployment (and further embedded types) easier to access - however having a generic variable name allowed for minimal changes to the file in general while moving from one resource to another - as these programs grow i have a hunch that our ability to keep as many resource specific code out of these files will be helpful 

what if we exported the “i” (for internal) to “K” for “kube”

Reply:
heck - you could even call it “Kube” or “KubeObject” or something

Reply:
I refactored this to a named type instead of embedded. (what's the opposite of embedded? ~shrug~ )